### ISSUE TITLE
Import format is able to change pylint result and hide E1121 error

### ISSUE THREAD

**Issue by pchomik** [PyCQA/pylint#861]
My environment:
- python 2.7.9
- pylint 1.5.4

I used this test to reproduce the problem:

```
from os import path
from os.path import exists


def error_is_reported():
    exists('/tmp', '/work')


def error_is_hidden():
    path.exists('/tmp', '/work')
```

**Comment by PCManticore** [comment]
I can't reproduce this outside of an virtualenv. The problem seems to be though that ``path.exists`` will be inferred as multiple values, which will make this call (https://github.com/PyCQA/pylint/blob/master/pylint/checkers/typecheck.py#L523) to return None, leading to ``path.exists`` node to be skipped entirely for this check. We're doing this in order to avoid false positives for things we don't understand, but in this particular case, we might improve the result, by returning the first non-Uninferable value from the inference results.

I have to think what implications this might have, so this might take a bit before I'll have a fix for this issue.

**Comment by pchomik** [comment]
Thanks for response and excellent explanation. I'm waiting for your investigation. In meantime I will double check other cases with same result.

**Comment by Pierre-Sassoulas** [comment]
Both messages are now reported with pylint 2.14
